example/javascriptlib/testing/1-test-suite/bar/test/src/bar/calculator.test.ts

```diff
@@ -1,4 +1,4 @@
-import {Calculator} from 'bar/calculator';
+import {Calculator} from '../../../src/calculator';
```

Comment:
This seems worse than the existing behavior. Why does only this example need to change?